### ISSUE TITLE
Experiments to Run

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#91]
This issue serves as a place to record a set of experiments that we would like to run.

- [ ] Test the effect of background randomization. Train the same network with and without background randomization. Perform this experiment with a multi-object dataset.

- [ ] Effect of descriptor dimension. So far we have been training with `D = 3`. Also test `D = 16` and `D = 32`. This experiment is currently in progress on the mulit_object_in_isolation dataset.

- [ ] Parameter sweeps. Parameters to tweak include learning_rate, weight_decay, non_match_loss_weight, etc.

- [ ] Do we get object consistency without explicitly enforcing it in the loss function?

- [x] Do multiple objects in isolation generalize to cluttered scenes?
     - **Answer: No**

Post the results of running such experiments below as comments.
